afdko/Tools/SharedData/FDKScripts/ProofPDF.py

```diff
@@ -776,15 +734,14 @@ def openFile(path, txPath):
 			data = ff.read(10)
 			ff.close()
 		except (IOError, OSError):
-			import traceback
 			traceback.print_exc()
 			raise FontError("Failed to open and read font file %s. Check file/directory permissions." % path)
 
 		if len(data) < 10:
 			raise FontError("Error: font file was zero size: may be a resource fork font, which this program does not process. <%s>." % path)
 		if (data[:4] == "OTTO") or (data[:4] == "true") or (data[:4] == "\0\1\0\0"): # it is an OTF/TTF font, can process file directly
 			try:
-				ttFont = ttLib.TTFont(path)
+				ttFont = TTFont(path)
 			except (IOError, OSError):
 				raise FontError("Error opening or reading from font file <%s>." % path)
 			except TTLibError:
```

Comment:
`TTLibError` is undefined -- see comment at top of file about need to import it from `fontTools.ttLib`